docs-src/source/tutorial/ppm-breakdown-comb.rst

```diff
@@ -212,7 +212,7 @@ Consider this contrived example:
 
 .. code-block:: DaeDaLus
 
-    def P = (Match1 'A') <| (^ 'B')
+    def P = ($['A']) <| (^ 'B')
```

Comment:
There's no need for parens on either side of the `<|`.   The `$[  ]` notation works like parens already.